tests/utils/test_sqlite.py

```diff
@@ -0,0 +1,149 @@
+from typing import Dict
+from typing import List
+
+import sqlalchemy as sa
+from sqlalchemy.engine.reflection import Inspector
+
+from spinta.utils.sqlite import migrate_table
+
+
+def test_create_table():
```

Comment:
Please add parametrization to all tests, to test both ways, with alter table and with copy table. For this to work, `migrate_table` should accept a `copy` argument if `copy=True`, then `_migrate_with_insert_from_select` is used, if `copy=False`, then `_migrate_with_alter_table` is used.

Reply:
Please use parameterization:

https://docs.pytest.org/en/6.2.x/parametrize.html

Instead of adding multiple copies of same tests.